wsi/drm/drmdisplay.h

```diff
@@ -160,6 +171,7 @@ class DrmDisplay : public PhysicalDisplay {
   uint32_t canvas_color_prop_ = 0;
   uint32_t connector_ = 0;
   bool dcip3_ = false;
+  uint32_t max_bpc_prop_ = 0;
```

Comment:
Does this need to be an "uint32_t"?  "uint8_t" is sufficient don't you think?

Reply:
Probably, but here I choose earlier implementations..